spec/spec_app/design/api.rb

```diff
@@ -35,4 +35,9 @@
     base_path "/"
   end
   
+  info '2.0' do
+    version using: :path
+    base_path '/'
```

Comment:
should we use an explicit base_path for this version? ... so see how it will show in the final path.
i.e. `base_path '/special_base'`